website/docs/reference/sdks/node.md

```diff
@@ -51,15 +50,14 @@ const { startUnleash } = require('unleash-client');
 const unleash = await startUnleash({
   url: 'https://YOUR-API-URL',
   appName: 'my-node-name',
-  environment: process.env.APP_ENV,
   customHeaders: { Authorization: 'SOME-SECRET' },
 });
 ```
 
   </TabItem>
 </Tabs>
 
-The example code above will initialize the client SDK, and connect to the Unleash-hosted demo instance. It also uses the API token for the demo instance. You should change the URL and the Authorization header (API token) with the correct values for your instance, which you may locate under “Instance admin” in the menu.
+The example code above will initialize the client SDK, and try to the Unleash instance you point it to. You will need to use your own Unleash instance URL and a [server-side API token](../api-tokens-and-client-keys.mdx#client-tokens) to connect. For steps on how to create an API token, refer to the [_how to create API tokens_](../../how-to/how-to-create-api-tokens.mdx) guide.
```

Comment:
Seems like a typo, should it be `try to connect to` instead?

Reply:
Yup! I caught that right after merging, so quick-added it in a commit d2245583eca4c76dc6f144a5f498955fd8c924ec 😅  Good catch 🙌🏼